src/eth/primitives/chain_id.rs

```diff
@@ -51,3 +51,132 @@ impl From<ChainId> for U256 {
         value.0.as_u64().into()
     }
 }
+// Creating a new ChainId instance with a valid U64 value
+#[test]
+fn create_chainid_with_valid_u64() {
```

Comment:
These types of tests would be better implemented as macros see https://github.com/cloudwalk/stratus/blob/main/src/ext.rs#L304